docs/code/Flakes/Explanation/Why.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Why Flakes?
+date: 2021-02-10
+---
+
+Nix Flakes are a new feature of the "nix package manager". They unify the typical ways of declaring inputs and ouputs of a project under a common syntax.
+
+But why use them?
+
+## Why?
+
+  - Unify the management of inputs (make redundant tools like `niv`).
+  - Remove sources of impurity that existed in the previous model.
+  - Standardize the output format of repositories.
```

Comment:
One other benefit of this new format is that it facilitates composability, so it becomes easier to connect one flake's outputs to other flake's inputs.